Release v1.0.0 (#140)
mpytools/filefisher: CHANGELOG.md

# Changelog

> ## v1.0.0 - 07.01.2025

Reviewer: ```suggestion
## v1.0.0 - 07.01.2025

```

> The package was renamed to filefisher (from filefinder). First release to pypi. Includes some modifications of the FileContainer class to be more user friendly. Defines and tests minimum supported versions of the dependencies and drops support for python 3.9, while testing explicitly on python 3.13. Moreover, adds documentation and some handy functionalities like concatenating two FileContainers or finding exactly one path or file.

Reviewer: ```suggestion
version 1.0.0 renames the package to filefisher (from filefinder) and makes the first release to pypi. It includes some modifications of the `FileContainer` class to be more user friendly, allows concatenating two `FileContainers` and adds methods to find exactly one path or file. Defines and tests minimum supported versions of the dependencies and adds documentation on readthedocs.```

# Changelog

## v1.0.0 - 07.01.2025

> version 1.0.0 renames the package to filefisher (from filefinder) and makes the first release to pypi. It includes some modifications of the `FileContainer` class to be more user friendly, allows concatenating two `FileContainers` and adds methods to find exactly one path or file. Defines and tests minimum supported versions of the dependencies and adds documentation on readthedocs.```

Author: ```suggestion
Version 1.0.0 renames the package to filefisher (from filefinder) and makes the first release to pypi. It includes some modifications of the `FileContainer` class to be more user friendly, allows concatenating two `FileContainers` and adds methods to find exactly one path or file. Defines and tests minimum supported versions of the dependencies and adds documentation on readthedocs.
```

Reviewer: No you can remove them
